src/main/kotlin/com/redhat/devtools/gateway/devworkspace/DevWorkspaces.kt

```
@@ -283,14 +283,15 @@ class DevWorkspaces(private val client: ApiClient) {
         timeout: Long, // in seconds
         checkCancelled: (() -> Unit)? = null
     ): Boolean {
-        return withTimeoutOrNull(timeout.seconds) {
+        @Suppress("ConvertLongToDuration")
+        return withTimeoutOrNull(timeout * 1000L) {
             while (true) {
                 checkCancelled?.invoke()
 
                 val devWorkspace = try {
                     DevWorkspaces(client).get(namespace, name)
                 } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)